recipe/meta.yaml

```diff
@@ -34,8 +34,12 @@ about:
   home: https://github.com/ethereum/eth-typing
   summary: 'eth-typing: Common type annotations for ethereum python packages'
   license: MIT
+  license_family: MIT
   license_file: LICENSE
+  doc_url: https://github.com/ethereum/eth-typing/blob/main/README.md
+  dev_url: https://github.com/ethereum/eth-typing
+  description: Common type annotations for ethereum python packages
 
 extra:
   recipe-maintainers:
-    - step21
+    - snegireff
```

Comment:
usually, we do not update maintainers unless it's a new feedstock or a total rewrite